Sources/Swift/Integrations/UserFeedback/SentryFeedback.swift

```diff
@@ -0,0 +1,63 @@
+@_implementationOnly import _SentryPrivate
+import Foundation
+
+@objcMembers
+class SentryFeedback: NSObject, SentrySerializable {
+    enum Source: String {
+        case widget
+        case custom
+    }
+
+    var name: String?
+    var email: String?
+    var message: String
+    var source: Source
+    let eventId: SentryId
+    
+    /// PNG data for the screenshot image
+    var screenshot: Data?
+    
+    /// The event id that this feedback is associated with, like a crash report.
+    var associatedEventId: String?
+
+    /// - parameter screenshot Image encoded as PNG data.
+    init(message: String, name: String?, email: String?, source: Source = .widget, associatedEventId: String? = nil, screenshot: Data? = nil) {
+        self.eventId = SentryId()
+        self.name = name
+        self.email = email
+        self.message = message
+        self.source = source
+        self.associatedEventId = associatedEventId
+        self.screenshot = screenshot
+        super.init()
+    }
+    
+    func serialize() -> [String: Any] {
+        var dict: [String: Any] = [
+            "message": message
+        ]
+        if let name = name {
+            dict["name"] = name
+        }
+        if let email = email {
+            dict["contact_email"] = email
+        }
+        if let associatedEventId = associatedEventId {
+            dict["associated_event_id"] = associatedEventId
+        }
+        dict["source"] = source.rawValue
+        
+        return dict
+    }
+    
+    /**
+     * - note: Currently there is only a single attachment possible, for the screenshot, of which there can be only one.
+     */
+    func attachments() -> [Attachment] {
```

Comment:
`l`: I think renaming this to something like `convertScreenshotToAttachment` would communicate the info in the comment a bit better. Now, when only looking at the code in the SentryClient I could assume that the SentryFeedback has multiple attachments. Only when I read this comment or jump to this code I understand what's going on.

Reply:
What difference does it make what kind of attachments it appends?

Reply:
As I mentioned above, when reading the SentryClient code, I was a bit confused that the feedback can have multiple attachments, so I jumped here and saw that it's one attachment. Renaming the method to something I proposed above, could make this a bit easier to understand. But we can ignore this. That's nitpicking. Let's focus on the more important things. It's not even worth the time I write this comment for discussing such small things 😅